src/lib/components/textarea/TextArea.component.tsx

```diff
@@ -2,22 +2,42 @@ import React, { forwardRef, HTMLProps } from 'react';
 import styled, { css } from 'styled-components';
 import { spacing } from '../../spacing';
 import { getTheme } from '../../utils';
-type Props = Omit<HTMLProps<HTMLTextAreaElement>, 'as'>;
+
+type TextAreaVariant = 'code' | 'text';
+type Props =
+  | Omit<HTMLProps<HTMLTextAreaElement>, 'as'> & { variant?: TextAreaVariant };
```

Comment:
Does this type include the width/height props already ?  If yes does their types complies with `CSSProperties['width'];` ?

Reply:
Yep it does and it comply with `CSSProperties['width']`.